core/server/data/migrations/versions/4.46/2022-04-13-12-57-add-uuid-column-to-newsletters.js

```diff
@@ -0,0 +1,8 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('newsletters', 'uuid', {
+    type: 'string',
+    maxlength: 36,
+    nullable: true,
```

Comment:
Should we be dropping the `nullable` on this field after we've populated it in the migration below?

Reply:
Does that mean we update the `schema` to have `nullable:false` and then just add a migration to drop it after ?

Reply:
I believe so 🤔 https://github.com/TryGhost/Ghost/pull/14568/files#diff-18e7c5d15a806349f8a27507183762f3ef9d201415934325f2058f7d2c858c6b